### ISSUE TITLE
Microsoft Accessibility Standard violation in popup link

### ISSUE THREAD

**Issue by wandyezj** [Microsoft/vscode#56586]
Issue Type: <b>Bug</b>

Popup with the message:

"We recommend switching to our new User Setup distribution of Code for Windows! Click here to learn more."

used "here" as a link. This is a violation of Microsoft Accessibility Standards. Links should have descriptive text titles. 

VS Code version: Code 1.26.0 (4e9361845dc28659923a300945f84731393e210d, 2018-08-13T16:29:31.933Z)
OS version: Windows_NT x64 10.0.17735

**Comment by dbaeumer** [comment]
Verified in code since I am already on user install.
